### PR TITLE
add kubeadm alpha phase kubeconfig option(--output)

### PR DESCRIPTION
**Pull request by ghost**
**What this PR does / why we need it**:
This PR will add --output option to `kubeadm alpha phase kubeconfig [command]`
This option can change output directory of kubeconfig.

Prev: kubeadm alpha phase kubeconfig admin
  we can get admin.conf in /etc/kubernetes/ .

This PR: kubeadm alpha phase kubeconfig admin --output /root
  we can get admin.conf in /root/ .

Of course, --output is optional.(default directory is /etc/kubernetes)

Note)
  I have no confidence about my test code.

Thank you.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Review by luxas — COMMENTED** [review]
Thanks for this PR!

Why is this needed? Can you please elaborate more on the use-cases here?

**Comment by shun-miyoshi-com** [comment]
I want to test my code of kubernetes components(kube-apiserver, kube-controller-manager, ...) with RBAC.(for contribution to kubernetes)
But, I am not interested in how to create PKI files.
So, I used the following commands. 

```
kubeadm alpha phase certs all
kubeadm alpha phase kubeconfig all

``` 

I want to use my local directory which is not /etc/kubernetes.
```kubeadm alpha phase certs ``` command have option to store in local directory (--cert-dir String)
But ```kubeadm alpha phase kubeconfig ``` command don't have option to change output directory.

Thus, I posted this PR.

**Comment by shun-miyoshi-com** [comment]
Is my usecase wrong case?
Even if it is true, I think that it is wrong to have only a way to use fixed PATH.

File: cmd/kubeadm/app/cmd/phases/kubeconfig.go [hunk 1]
```diff
@@ -124,6 +124,7 @@ func getKubeConfigSubCommands(out io.Writer, outDir, defaultKubernetesVersion st
 		cmd.Flags().StringVar(&cfg.CertificatesDir, "cert-dir", cfg.CertificatesDir, "The path where certificates are stored.")
 		cmd.Flags().StringVar(&cfg.API.AdvertiseAddress, "apiserver-advertise-address", cfg.API.AdvertiseAddress, "The IP address or DNS name the API Server is accessible on.")
 		cmd.Flags().Int32Var(&cfg.API.BindPort, "apiserver-bind-port", cfg.API.BindPort, "The port the API Server is accessible on.")
+		cmd.Flags().StringVar(&outDir, "output", outDir, "The path where to save and store the kubeconfig file.")
```
Comment (luxas) [inline cmd/kubeadm/app/cmd/phases/kubeconfig.go]: I would like this to be `kubeconfig-dir` please, that is more descriptive I think

**Review by luxas — COMMENTED** [review]
/approve no-issue

Ok, thanks for the rationale. That is ok for me then.
Please fix the nit and I'll LGTM

File: cmd/kubeadm/app/cmd/phases/kubeconfig.go (hunk 1 above)
Comment (shun-miyoshi-com) [reply cmd/kubeadm/app/cmd/phases/kubeconfig.go]: I'm sorry, I missed your comment.
I fixed my PR.

**Comment by shun-miyoshi-com** [comment]
I'm sorry, I missed @luxas comment.
I fixed my PR.

**Review by luxas — APPROVED** [review]
/lgtm
/approve no-issue

**Pull request merged**